src/node/quote.cpp

```diff
@@ -6,32 +6,84 @@
 #include "ccf/pal/attestation.h"
 #include "ccf/service/tables/code_id.h"
 #include "ccf/service/tables/snp_measurements.h"
+#include "ccf/service/tables/uvm_endorsements.h"
+#include "node/uvm_endorsements.h"
 
 namespace ccf
 {
+  bool verify_enclave_measurement_against_uvm_endorsements(
+    kv::ReadOnlyTx& tx,
+    const CodeDigest& quote_measurement,
+    const std::vector<uint8_t>& uvm_endorsements)
+  {
+    auto uvm_endorsements_data =
+      verify_uvm_endorsements(uvm_endorsements, quote_measurement);
+    auto uvmes = tx.ro<SnpUVMEndorsements>(Tables::NODE_SNP_UVM_ENDORSEMENTS);
```

Comment:
It's weird to have SnpUVM, I think we can either have SnpUvm, or SNPUVM, but the mix seems quite inconsistent.